makefile

```
@@ -114,8 +126,14 @@ SO_EXT = $(if $(NOTMAC),so,dylib)
 # Default multisampling mode and multisampling line-width
 GLVIS_MULTISAMPLE  ?= 4
 GLVIS_MS_LINEWIDTH ?= $(if $(NOTMAC),1.4,0.01)
```

```suggestion
GLVIS_MS_LINEWIDTH ?= $(if $(NOTMAC),1.4,1.0)
```

With this version 0.01 is way too small.

---

Don't forget to update the CMake file too.